csrc/scheduler/reduction_utils.cpp

```diff
@@ -30,9 +31,11 @@ TensorView* scheduleReductionTV(
   // Inner here though is only relative to the other axis. When
   // rparams.fastest_dim == false, the reduction axis is logically outside the
   // iteration axis.
-  const int iter_axis = 0;
+  const int has_outmost_dim_sharded = isSharded(reduction_tv);
```

Comment:
nit: we use "outer" more often than "out" when referring to positions in a domain.

```suggestion
  const int has_outermost_dim_sharded = isSharded(reduction_tv);
```